ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/PeerDiscoveryTestHelper.java

```diff
@@ -145,7 +145,7 @@ public MockPeerDiscoveryAgent startDiscoveryAgent(
 
   public MockPeerDiscoveryAgent startDiscoveryAgent(final AgentBuilder agentBuilder) {
     final MockPeerDiscoveryAgent agent = createDiscoveryAgent(agentBuilder);
-    agent.start();
+    agent.start(65500).join();
```

Comment:
You could use `PeerDiscoveryTestHelper.nextAvailablePort` here.  Or you could try grabbing the discovery port from the agent's config and reusing that for the tcp port.  Would be nice if we don't have to start scattering random numbers throughout the tests.

Reply:
Done.

Reply:
Is it possible to clean up the rest of the magic numbers?

Reply:
I'll have a look.  They are just arbitrary numbers since we don't assert on them and don't open a port for them anywhere.

Reply:
I've pulled them out as constants - one of the tests already had a constant it had used for this previously.